src/static/css/font.css

```diff
@@ -0,0 +1,17 @@
+/* martel-sans-regular - latin */
+@font-face {
+    font-display: swap;
+    font-family: "Martel Sans";
+    font-style: normal;
+    font-weight: 400;
+    src: url("../fonts/martel-sans-v12-latin-regular.woff2") format("woff2");
+}
+
+/* mulish-regular - latin */
+@font-face {
+    font-display: swap;
+    font-family: Mulish;
+    font-style: normal;
+    font-weight: 400;
+    src: url("../fonts/mulish-v12-latin-regular.woff2") format("woff2");
+}
```

Comment:
I think we will need some additional weights and italic combinations for both of these. See the typestack (https://github.com/inclusive-design/idg-design-system/issues/16) in Figma.

Reply:
Thanks @greatislander, I added some more font weights as well as italic for Mulish font. I couldn't find italics for Martel on Google Fonts.